platform/windows/detect.py

```diff
@@ -209,14 +209,15 @@ def configure(env):
     env.Append(CPPPATH=['#platform/windows'])
 
     # Targeted Windows version: Vista (and later)
-    env.Append(CPPFLAGS=['-D_WIN32_WINNT=0x0600'])
+    winver = "0x0600" # Windows Vista is the minimum target for windows builds
 
     env['is_mingw'] = False
     if (os.name == "nt" and os.getenv("VCINSTALLDIR")):
         # build using visual studio
         env['ENV']['TMP'] = os.environ['TMP']
         env.Append(CPPPATH=['#platform/windows/include'])
         env.Append(LIBPATH=['#platform/windows/lib'])
+        env.Append(CCFLAGS=['/DWINVER=%s' % winver, '/D_WIN32_WINNT=%s' % winver])
```

Comment:
From what I read in https://blogs.msdn.microsoft.com/oldnewthing/20070411-00/?p=27283 and MSDN, it seems that it's not necessary to define WINVER when you already define _WIN32_WINNT.  This seems to be confirmed when you read `<sdkddkver.h>` (at least that of MinGW-w64) where _WIN32_WINNT is used as base to define all the others if they're not already defined.

But then, it doesn't hurt much either if you think it's better to define both.